catalog/app/containers/Bucket/Qurator/Section.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+
+import { ChatSkeleton } from 'components/Chat'
+import * as Model from 'model'
+
+import PageSection from '../Section'
+
+const QuratorSummary = React.lazy(() => import('./Summary'))
+
+interface QuratorSectionProps {
+  handle: Model.S3.S3ObjectLocation
+}
+
+export default function QuratorSection({ handle }: QuratorSectionProps) {
+  const children = React.useCallback(
```

Comment:
i don't think it makes a lot of sense to memoize this, just passing the callback directly is ok